docs/GettingStarted.md

```diff
@@ -175,6 +175,47 @@ Net::HTTP module.
 
     content = Net::HTTP.get(URI('http://127.0.0.1/index.html'))
 
+### Sidekiq
+
+The Sidekiq integration is a server-side middleware which will trace job
+executions. It can be added as any other Sidekiq middleware:
+
+    require 'sidekiq'
+    require 'ddtrace'
+    require 'ddtrace/contrib/sidekiq/tracer'
+
+    Sidekiq.configure_server do |config|
+      config.server_middleware do |chain|
+        chain.add(Datadog::Contrib::Sidekiq::Tracer, debug: true)
+      end
+    end
+
+#### Configure the tracer
+
+To modify the default configuration, simply pass arguments to the middleware.
+For example, to change the default service name and activate the debug mode:
+
+    Sidekiq.configure_server do |config|
```

Comment:
Actually I'm wondering what happen when we use Sidekiq with Rails. The current status is that if we install both Sidekiq and Rails, when Sidekiq is executed via ``bundle exec ...`` the Rails instrumentation is activated and Sidekiq Redis calls are traced (even the `x seconds` polling). Is it the expected behavior? I think that this middleware must be the only way to instrument the library so we may investigate and remove the side-effect of Rails instrumentation.

Also I was thinking about providing a good pattern (I think docs are enough) to instrument Sidekiq inside the Rails initializer so that our tracing configurations are all together. In this way Sidekiq is still separated from Rails instrumentation but users can easily follow our docs to share the Rails configuration with Sidekiq.